pinot-core/src/main/java/com/linkedin/pinot/core/operator/aggregation/function/AvgAggregationFunction.java

```diff
@@ -153,11 +155,66 @@ public AvgPair merge(@Nonnull AvgPair intermediateResult1, @Nonnull AvgPair inte
   @Nonnull
   @Override
   public Double extractFinalResult(@Nonnull AvgPair intermediateResult) {
-    long count = intermediateResult.getSecond();
-    if (count == 0L) {
+    if (intermediateResult._count == 0L) {
       return DEFAULT_FINAL_RESULT;
     } else {
-      return intermediateResult.getFirst() / count;
+      return intermediateResult._sum / intermediateResult._count;
+    }
+  }
+
+  // TODO: after removing the DataTableJavaSerDe, remove the Serializable.
+  public static class AvgPair implements Comparable<AvgPair>, Serializable {
```

Comment:
Will this class be reference outside AvgAggergationFunction? If so, may be it should be moved out of aggr function class.